WebRoot/js/ajax/dwt/xforms/XFormItem.js

```diff
@@ -995,10 +995,21 @@ XFormItem.prototype.handleKeyPressDelay = function(ev, domItem) {
 
 XFormItem.prototype.getKeyPressHandlerHTML = function () {
 
-	var keydownEv = "onkeydown";
-	if (AjxEnv.isNav) {
-		keydownEv = "onkeypress";
-	}
+        // if AjxEnv.isNav, etc.
+        var keydownEv = "onkeypress";
```

Comment:
Not sure what you mean, but DwtEvent.ONKEYPRESS has different event function associated with it.
"onkeypress"/"onkeydown" attributes  are for AjxEnv specific and have their own event functions.